internal/cmd/open.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+
+	"cdr.dev/coder-cli/coder-sdk"
+)
+
+func openCmd() *cobra.Command {
+	var (
+		workspaceName string
+		user          string
+	)
+
+	cmd := &cobra.Command{
+		Use:     "open",
+		Short:   "Open Coder workspace IDE",
+		Long:    "Open Coder workspace IDE in your default browser",
+		Example: "coder open --workspace dev",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			client, err := newClient(ctx, true)
+			if err != nil {
+				return err
+			}
+
+			if workspaceName == "" {
+				return xerrors.Errorf("Missing required parameter: --workspace")
+			}
+
+			workspaces, err := getWorkspaces(ctx, client, user)
+			if err != nil {
+				return err
+			}
+
+			workspaceID := ""
+			for _, w := range workspaces {
+				if w.Name == workspaceName {
+					workspaceID = w.ID
+				}
+			}
+			if workspaceID == "" {
+				return xerrors.Errorf("Workspace %q not found", workspaceName)
+			}
+
+			u := client.BaseURL()
+			u.Path += "/app/code"
```

Comment:
Ah, I guess this only works with VSCode and not JetBrains etc? I guess if we maintain this pattern in the future, we could add an --app flag to control the exact URL we use?